### DISCUSSION TITLE
FFMPEG Package

### DISCUSSION THREAD

**Discussion by AdiAbuAli** [mrlt8/docker-wyze-bridge#216] [Q&A] [answered]
Hi, 
Is FFMPEG installed as dependency in Wyze-Bridge docker or part of other packages (e.g. Wyzecam or RTSP-Simple_Server,...etc.).

Am asking beucase I have it when I install Wyze-Bridge, but the version is 3.12.x while latest is 4.4.1...?

**Comment by mrlt8** [comment, marked as answer]
Starting with [v0.7.0](https://github.com/mrlt8/docker-wyze-bridge/releases/tag/v0.7.0), we've been using [ffmpeg-for-homebridge](https://github.com/homebridge/ffmpeg-for-homebridge)
